src/Ormolu/Printer/Meat/Module.hs

```diff
@@ -39,7 +35,7 @@ p_hsModule loc@(L moduleSpan hsModule) = do
       Nothing -> pure ()
       Just hsmodName' -> line $ do
         located hsmodName' p_hsmodName
-        forM_ hsmodDeprecMessage (located' p_warningTxt)
+        forM_ hsmodDeprecMessage (located' (p_warningTxt ModuleWarning))
```

Comment:
Since you seem to only be using `p_warningTxt` and `ModuleWarning` here (outside of the `Declaration.Warning` module), I'd be tempted to expose a single `p_warningModule` function instead. But it's not really an issue either way :sweat_smile: